basic-configuration.md

```diff
@@ -1,121 +1,128 @@
-priced.
-
-To access the catalog configuration, follow these steps:
-
-1. Log in to the Magento 2 admin panel.
-2. Navigate to **Stores** > **Configuration**.
-
-### Example: Setting Up Product Categories
-
-Product categories help customers navigate your store and find the products they are looking for. To set up product
-categories:
-
-1. Go to **Stores** > **Configuration**.
-2. In the left-hand menu, click on **Catalog** > **Catalog**.
-3. Expand the **Category Top Navigation** section.
-4. Enable the **Enable Category Top Navigation Menu** option.
-5. Click on **Save Config**.
-
-## Payment Configuration
-
-Payment configuration settings in Magento 2 allow you to set up and manage the payment methods available to your
-customers.
-
-To access the payment configuration, follow these steps:
-
-1. Log in to the Magento 2 admin panel.
-2. Navigate to **Stores** > **Configuration**.
-
-### Example: Enabling PayPal Express Checkout
-
-PayPal Express Checkout is a popular payment method. To enable PayPal Express Checkout:
-
-1. Go to **Stores** > **Configuration**.
-2. In the left-hand menu, click on **Sales** > **Payment Methods**.
-3. Expand the **PayPal Express Checkout** section.
-4. Set the **Enabled** option to **Yes**.
-5. Enter your PayPal API credentials in the appropriate fields.
-6. Click on **Save Config**.
-
-## Shipping Configuration
-
-Shipping configuration settings in Magento 2 allow you to manage how shipping is calculated and handled in your store.
-
-To access the shipping configuration, follow these steps:
-
-1. Log in to the Magento 2 admin panel.
-2. Navigate to **Stores** > **Configuration**.
-
-### Example: Configuring Flat Rate Shipping
-
-Flat rate shipping charges a fixed rate for shipping regardless of the order's weight or location. To configure flat
-rate shipping:
-
-1. Go to **Stores** > **Configuration**.
-2. In the left-hand menu, click on **Sales** > **Shipping Methods**.
-3. Expand the **Flat Rate** section.
-4. Set the **Enabled** option to **Yes**.
-5. Configure the **Title**, **Method Name**, **Price**, and other relevant fields.
-6. Click on **Save Config**.
-
-## Conclusion
-
-In this documentation, we have covered the basic configuration settings in Magento 2, including system, store, catalog,
-payment, and shipping configurations. By following the provided instructions and using the code snippets and examples,
-you can easily configure your Magento 2 store to meet your specific requirements. For more advanced configuration
-options, refer to the official Magento 2 documentation or consult with a Magento developer.
+## Introduction
+
+Magento 2, a versatile eCommerce platform, offers extensive configurability to meet the needs of developers. This
+document discusses the fundamental configuration in Magento 2, focusing on `app/etc/env.php`, `app/etc/config.php`,
+`core_config_data` table, `config.xml`, and managing configuration programmatically.
```

Comment:
The ENV variable config option is missing in the list here.

Reply:
@DavidLambauer could you add this small missing part so that we can review and merge this PR? thank you!